AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientObserver.swift

```diff
@@ -37,6 +39,12 @@ class DefaultAudioClientObserver: NSObject, AudioClientDelegate {
         let newAudioState = Converters.AudioClientState.toSessionStateControllerAction(state: audioClientState)
         let newAudioStatus = Converters.AudioClientStatus.toMeetingSessionStatusCode(status: status)
 
+        if newAudioState == .unknown {
+            logger.info(msg: "AudioClient State: \(newAudioState) Unknown Status rawValue: \(status.rawValue)")
```

Comment:
Do we want to log `audioClientState`'s raw value as well?

Reply:
The only one that we grouped are the .fail, but I can update.